src/backend/interfaces/REST/go.server/middlewares/authentication.go

```diff
@@ -5,16 +5,61 @@
 package http_middleware
 
 import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/sha256"
+	"encoding/asn1"
 	"encoding/base64"
+	"errors"
+	"log"
```

Comment:
we make use of `logrus` to log, not standard pkg `log`. Replace this import by  
 `log "github.com/Sirupsen/logrus"`

Reply:
ok done